packages/site/editorMobileBundle.rollup.config.js

```diff
@@ -28,38 +29,43 @@ export default {
     }),
     alias({
       entries: [
-        { find: "react-native", replacement: "./src/MobileOverrides.js" },
+        { find: "react-native", replacement: "./src/MobileOverrides.jsx" },
 
         {
           find: "react-native-keyboard-aware-scroll-view",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-toast-message",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-safe-area-context",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-gesture-handler",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-modalize",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-svg",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
         {
           find: "react-native-reanimated",
-          replacement: path.resolve(dirname, "./src/MobileOverrides.js"),
+          replacement: path.resolve(dirname, "./src/MobileOverrides.jsx"),
         },
       ],
     }),
+    babel({
```

Comment:
We're now bundling our MobileOverrides file so we can get JSX in there. Still no Typescript because I don't want the added complexity for a single file.